14/umbraco-cms/extending/package-manifest.md

```diff
@@ -44,40 +38,173 @@ Before Umbraco 14, the manifest was declared in a `package.manifest` file instea
+        "scopes": {
+            "": ""
+        }
+    },
     "extensions": []
 }
 ```
 
+### Id
+
+The unique identifier for your package. This is used to identify your package and should be unique to your package. If you are creating a package that is distributed via NuGet, you should use the NuGet package ID as the ID for your package.
+
 ### Name
 
 Allows you to specify a friendly name for your package that will be used for telemetry. If no name is specified the name of the folder will be used instead.
 
 ### Version
 
-The version of your package, if this is not specified there will be no version-specific information for your package.
+The version of your package, if this is not specified there will be no version-specific information for your package. This is used for telemetry and to help users understand what version of your package they are using. It is also used for package migrations. The version should follow the [Semantic Versioning](https://semver.org/) format.
 
 ### Allow Package Telemetry
 
 With this field, you can control the telemetry of this package, this will provide Umbraco with the knowledge of how many installations use this package.
 
-## Extensions
+Default is `false`.
+
+### Allow Public Access
+
+This field is used to allow public access to the package. If set to `true`, the package will be available for anonymous usage, for example on the login screen. If set to `false`, the package will only be available to logged-in users.
+
+Default is `false`.
 
-The `extensions` field is an array of UI Extension Manifests, each Manifest describes a single UI Extension. You can read more about this in the [UI Extension Types](property-editors/broken-reference/) article.
+### Importmap
+
+The `importmap` field is an object that contains two properties: `imports` and `scopes`. This is used to define the import map for the package. The `imports` property is an object that contains the import map for the package. The `scopes` property is an object that contains the scopes for the package.
+
+**Example**
+
+This example shows how to define an import map for a module exported by a package:
+
+{% code title="umbraco-package.json" lineNumbers="true" %}
+
+```json
+{
+    "importmap": {
+        "imports": {
+            "mypackage/services": "/App_Plugins/MyPackage/services/index.js",
+        }
+    }
+}
+```
+
+{% endcode %}
+
+The `imports` object contains the import map for the package. The key is the specifier for the module that is being imported, and the value is the URL of the module.
+
+This allows developers to consume modules that are exported by a package without having to know the exact path to the module:
+
+{% code title="index.js" %}
+
+```javascript
+import { MyService } from 'mypackage/services';
+```
+
+{% endcode %}
+
+Umbraco supports the current specification of the property as outlined on MDN Web Docs: [importmap](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap).
+
+### Extensions
```

Comment:
As there is so much text for each of these extensions, do you think it would make sense to add it to a table instead?

I think it's a little hard to get an overview of these lists: 
![image](https://github.com/umbraco/UmbracoDocs/assets/24529086/4e5a4afd-5a9d-4528-b035-1c9c6dc0b668)

At least in a table, the extensions would be in one column, and it would be a bit easier to find what you're looking for.

What do you think?

Reply:
Yes, it makes sense to use a table instead. As I only saw the raw markdown, a table looked more confusing than a list, but it makes sense on GitBook to do a table. I'll see if I can be clever with some search/replace.